src/js/structs/Service.js

```diff
@@ -146,6 +146,10 @@ module.exports = class Service extends Item {
     let deployments = this.getDeployments();
     let queue = this.getQueue();
 
+    if (this.getInstancesCount() === 0 && deployments.length === 0) {
```

Comment:
We need to also check against the amount of running tasks. From the docs: 

> An application with a target instances of 0 and whose running tasks count is 0. True when the JSON returned looks like: app.instances === 0 && app.tasksRunning === 0

https://mesosphere.github.io/marathon/docs/marathon-ui.html